x-pack/plugins/rollup/public/crud_app/sections/job_list/job_table/job_table.js

```diff
@@ -379,33 +372,27 @@ export class JobTableUi extends Component {
 
         <EuiSpacer size="m" />
 
-        {jobs.length > 0 ? (
```

Comment:
We should preserve this check, so we can display feedback when the user has entered search input which yields no matching jobs. See https://github.com/elastic/kibana/pull/31642 for some more context.

Reply:
OK I see, will put it back